lua/telescope/make_entry.lua

```diff
@@ -294,15 +294,25 @@ function make_entry.gen_from_symbols(opts)
   opts = opts or {}
   opts.tail_path = get_default(opts.tail_path, true)
 
-    local displayer = entry_display.create {
+  local displayer = entry_display.create {
     separator = "",
+    hl_chars = { ['['] = 'TelescopeBorder', [']'] = 'TelescopeBorder' },
```

Comment:
@sunjon Thats the new interface. Any comments?

Reply:
Seems reasonable. I've used it in the new LSP layout for inline fileinfo.

Reply:
Saw it :)